controllers/submariner/gateway_resources.go

```diff
@@ -199,6 +199,7 @@ func newGatewayPodTemplate(cr *v1alpha1.Submariner, name string, podSelectorLabe
 						{Name: "SUBMARINER_HEALTHCHECKENABLED", Value: strconv.FormatBool(healthCheckEnabled)},
 						{Name: "SUBMARINER_HEALTHCHECKINTERVAL", Value: strconv.FormatUint(healthCheckInterval, 10)},
 						{Name: "SUBMARINER_HEALTHCHECKMAXPACKETLOSSCOUNT", Value: strconv.FormatUint(healthCheckMaxPacketLossCount, 10)},
+						{Name: "SUBMARINER_GWMETRICSPORT", Value: gatewayMetricsServerPort},
```

Comment:
Are we changing these to `SUBMARINER_METRICS_PORT` or not?

Reply:
@mkolesnik Yeah. I have the changes ready. Will push once get power and internet back.